server/resources/resources.go

```diff
@@ -62,7 +62,7 @@ func setSysRAMBytes() {
 	}
 	mem := gosigar.Mem{}
 	mem.Get()
-	allocatedRAMBytes = int64(mem.ActualFree)
+	allocatedRAMBytes = int64(mem.Total)
```

Comment:
this LGTM, but a slightly better default for darwin might involve subtracting the current `wired memory` usage. That value seems to hover at around 1.5 - 2GB and can't be swapped to disk unlike other types of memory.